Marlin/src/lcd/menu/menu_main.cpp

```diff
@@ -503,7 +503,7 @@ void menu_main() {
     });
   #endif
 
-  #if ENABLED(ADVANCED_PAUSE_FEATURE) && DISABLED(DISABLE_ENCODER)
+  #if ALL(ADVANCED_PAUSE_FEATURE, FILAMENT_CHANGE_MENU_AT_BOTTOM)
```

Comment:
Did you intend to remove `DISABLED(DISABLE_ENCODER)` from this item? I don't exactly recall the rationale for `DISABLE_ENCODER` affecting this particular item (along with `BACK_ITEM`), except that it is optional with `TOUCH_SCREEN`.

Reply:
It's actually pointless since scrolling up to get to last menu item doesnt work. Filchange must be always somewhere at the top.

Reply:
You can still rapid-scroll down and it will stop at the bottom—it's still much faster to access the last item than one somewhere in the middle. And heavy use of filchange is not as universal as you think—I would rather have it in the middle, in a submenu, or absent entirely.

Reply:
Yep, for direct extruder i only use preheat, option to remove filchange form main menu would be neat, and another one to skip all unload/load steps and leave only resume prompt.